### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by sa-shiro** [adieyal/sd-dynamic-prompts#196]
**Describe the bug**
Dynamic Prompts dont work in the latest sd release (running on google colab)

**To Reproduce**
Steps to reproduce the behaviour, include the prompt you used if applicable:
1. run colab script till webui is accessible
2. put "photo of a {blue|red} ball" in prompt and generate
3. see error

Also mention which checkboxes have been set.

**Are you using a local install or a colab?**
- Google Colab https://github.com/ddPn08/automatic1111-colab

**Expected behaviour**
A random image being generated

console log:
```log
Error completing request
Arguments: ('task(2plxy3nlbavh9jo)', 'photo of a {blue|red} ball', '', [], 20, 0, False, False, 1, 1, 7, -1.0, -1.0, 0, 0, 0, False, 512, 512, False, 0.7, 2, 'Latent', 0, 0, 0, 0, True, False, 1, False, False, False, 1.1, 1.5, 100, 0.7, False, False, True, False, False, 0, False, False, False, False, '', 1, '', 0, '', True, False, False) {}
Traceback (most recent call last):
  File "/content/stable-diffusion-webui/modules/call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "/content/stable-diffusion-webui/modules/call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "/content/stable-diffusion-webui/modules/txt2img.py", line 52, in txt2img
    processed = process_images(p)
  File "/content/stable-diffusion-webui/modules/processing.py", line 475, in process_images
    res = process_images_inner(p)
  File "/content/stable-diffusion-webui/modules/processing.py", line 574, in process_images_inner
    file.write(processed.infotext(p, 0))
  File "/content/stable-diffusion-webui/modules/processing.py", line 327, in infotext
    return create_infotext(p, self.all_prompts, self.all_seeds, self.all_subseeds, comments=[], position_in_batch=index % self.batch_size, iteration=index // self.batch_size)
  File "/content/stable-diffusion-webui/modules/processing.py", line 457, in create_infotext
    negative_prompt_text = "\nNegative prompt: " + p.all_negative_prompts[index] if p.all_negative_prompts[index] else ""
IndexError: list index out of range
```

**Comment by ArrowM** [comment]
dup of #195

**Comment by adieyal** [comment]
Closing this issue - it has been addressed in #195.
